versioned_docs/version-v0.11.0/int/quickstart/group/quickstart-group-cli.md

```diff
@@ -0,0 +1,220 @@
+- Decide who the leader of your cluster will be. Only leaders have to perform [step 2](#step-2-leader-creates-the-dkg-configuration-file-and-distributes-it-to-everyone-else) and [step 5](#step-5-activate-the-deposit-data) in the quickstart process. They do not get any special privilege.
+
+## Step 1. Creating and backing up a private key for charon
+
+The first step of running a cluster is preparing for a distributed key generation ceremony. To do this everyone must create an [ENR](../../faq/errors.mdx#what-is-an-enr) for their charon client. This ENR is a public/private key pair, and allows the other charon clients in the DKG to identify and connect to your node.
+
+```sh
+# Clone this repo
+git clone https://github.com/ObolNetwork/charon-distributed-validator-node.git
+
+# Change directory
+cd charon-distributed-validator-node
+
+# Create your charon ENR private key, this will create a charon-enr-private-key file in the .charon directory
+docker run --rm -v "$(pwd):/opt/charon" obolnetwork/charon:v0.11.0 create enr
+```
+
+You should expect to see a console output like
+
+    Created ENR private key: .charon/charon-enr-private-key
+    enr:-JG4QGQpV4qYe32QFUAbY1UyGNtNcrVMip83cvJRhw1brMslPeyELIz3q6dsZ7GblVaCjL_8FKQhF6Syg-O_kIWztimGAYHY5EvPgmlkgnY0gmlwhH8AAAGJc2VjcDI1NmsxoQKzMe_GFPpSqtnYl-mJr8uZAUtmkqccsAx7ojGmFy-FY4N0Y3CCDhqDdWRwgg4u
+
+> ⚠️ Attention
+>
+> Please make sure to create a backup of the private key at `.charon/charon-enr-private-key`. Be careful not to commit it to git! **If you lose this file you won't be able to take part in the DKG ceremony.**
+
+If you are taking part in an organised Obol testnet, submit the created ENR public address (the console output starting with `enr:-...` not the contents of the private key file) to the appropriate typeform.
+
+## Step 2. Leader creates the DKG configuration file and distributes it to cluster operators
+
+The leader will prepare the `cluster-definition.json` file for the Distributed Key Generation ceremony using the `charon create dkg` command.
+
+In future, step 1 and step 2 of this guide will use the [Obol Distributed Validator Launchpad](https://docs.obol.tech/docs/dvk/distributed_validator_launchpad) to facilitate and verify these files are created in an authenticated manner.
+
+```
+# Prepare an environment variable file
+cp .env.create_dkg.sample .env.create_dkg
+
+# Populate the .env.create_dkg file with the cluster name, the fee recipient and withdrawal Ethereum addresses and the 
+# operator ENRs of all the operators participating in the DKG ceremony.
+
+# Run the `charon create dkg` command that generates DKG cluster-definition.json file.
+docker run --rm -v "$(pwd):/opt/charon" --env-file .env.create_dkg obolnetwork/charon:v0.11.0 create dkg
+```
+
+This command should output a file at `.charon/cluster-definition.json`. This file needs to be shared with the other operators in a cluster.
```

Comment:
@OisinKyne should we switch this step from `charon create dkg` to the `dv-launchpad` entirely this time?